charts/fluent-bit/templates/_pod.tpl

```diff
@@ -1,4 +1,5 @@
 {{- define "fluent-bit.pod" -}}
+{{- include "fluent-bit.automountServiceAccountToken" . }}
```

Comment:
I think this would be clearer if it wasn't a template; this is a case where KISS > DRY.